src/Mod/Import/App/ifc4.py

```diff
@@ -26588,25 +26588,25 @@ def fset( self, value ):
 class ifctasktimerecurring(ifctasktime):
 	'''Entity ifctasktimerecurring definition.
 
-	:param recurrance
-	:type recurrance:ifcrecurrencepattern
+	:param recurrence
```

Comment:
right, i guess i didn't realize that it would change the API. I understood that I was changing source code but thought it wouldn't have repercussions. 
How do i retract this commit? 
I'll sit on this change and open a separate ticket about it. Though I wonder how much of an issue it would be to change the API?